src/test/java/BuyTshirtProcessTest.java

```diff
@@ -31,26 +41,33 @@ public void setup() {
     options.addArguments("--disable-dev-shm-usage");
     options.addArguments("--headless");
     driver = new ChromeDriver(options);
+    catalog = new SaucedomoCatalogPage(driver);
+    checkOut = new SaucedomoCheckOutPage(driver);
+    logIn = new SaucedomoLogInPage(driver);
+    shoppingCart = new SaucedomoShoppingCartPage(driver);
+    verification = new SaucedomoVerificationPage(driver);
   }
 
   @Test(description = "This test is in charge of navigating through the webpage and"
       + "simulating the process of acquisition for a t-shirt")
   public void buyTshirtTest() {
-    driver.get("https://www.saucedemo.com/");
-    driver.findElement(By.id("user-name")).sendKeys("standard_user");
-    driver.findElement(By.id("password")).sendKeys("secret_sauce");
-    driver.findElement(By.id("login-button")).click();
-    driver.findElement(By.id("add-to-cart-sauce-labs-bolt-t-shirt")).click();
-    driver.findElement(By.id("shopping_cart_container")).click();
-    driver.findElement(By.id("checkout")).click();
-    driver.findElement(By.id("first-name")).sendKeys("Juan");
-    driver.findElement(By.id("last-name")).sendKeys("Palma");
-    driver.findElement(By.id("postal-code")).sendKeys("77901");
-    driver.findElement(By.id("continue")).click();
-    driver.findElement(By.id("finish")).click();
-    Assert.assertNotNull(driver.findElement(By.id("checkout_complete_container")));
-    Assert.assertTrue(driver.findElement(By.id("checkout_complete_container"))
-        .findElement(By.className("complete-header")).getText().contains("THANK YOU"));
+    logIn.visitLogin();
+    logIn.logIn("standard_user", "secret_sauce");
+
+    catalog.selectItem();
+    catalog.goToShoppingCart();
+
+    shoppingCart.goToCheckOut();
+
+    checkOut.fillOutForm("Juan", "Palma", "77601");
+
+    verification.finishProcess();
+
+    Object container = driver.findElement(By.id("checkout_complete_container"));
+    String result = driver.findElement(By.id("checkout_complete_container"))
+        .findElement(By.className("complete-header")).getText();
```

Comment:
Into a test you should not call locators, I mean, all locators should encapsulated into page object models